Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -26,32 +32,43 @@ public class ImportCommand : CodeExplorerCommandBase
 
         private readonly IVBE _vbe;
         private readonly IFileSystemBrowserFactory _dialogFactory;
-        private readonly IList<string> _importableExtensions;
-        private readonly string _filterExtensions;
         private readonly IParseManager _parseManager;
+        private readonly IProjectsProvider _projectsProvider;
+        private readonly IModuleNameFromFileExtractor _moduleNameFromFileExtractor;
+        private readonly IDictionary<ComponentType, IRequiredBinaryFilesFromFileNameExtractor> _binaryFileExtractors;
+        private readonly IFileExistenceChecker _fileExistenceChecker;
 
+        protected readonly IDeclarationFinderProvider DeclarationFinderProvider;
         protected readonly IMessageBox MessageBox;
 
         public ImportCommand(
             IVBE vbe,
             IFileSystemBrowserFactory dialogFactory,
             IVbeEvents vbeEvents,
             IParseManager parseManager,
+            IDeclarationFinderProvider declarationFinderProvider,
+            IProjectsProvider projectsProvider,
+            IModuleNameFromFileExtractor moduleNameFromFileExtractor,
+            IEnumerable<IRequiredBinaryFilesFromFileNameExtractor> binaryFileExtractors,
+            IFileExistenceChecker fileExistenceChecker,
             IMessageBox messageBox)
             : base(vbeEvents)
         {
             _vbe = vbe;
             _dialogFactory = dialogFactory;
             _parseManager = parseManager;
+            _projectsProvider = projectsProvider;
+            DeclarationFinderProvider = declarationFinderProvider;
```

Comment:
this sticks out like a sore thumb here. Can we move it down to the other protected and public properties?